functions/Get-DbaDbView.ps1

```diff
@@ -105,14 +153,14 @@ function Get-DbaDbView {
             }
 
             $defaults = 'ComputerName', 'InstanceName', 'SqlInstance', 'Database', 'Schema', 'CreateDate', 'DateLastModified', 'Name'
-            foreach ($view in $views) {
+            foreach ($v in $views) {
 
-                Add-Member -Force -InputObject $view -MemberType NoteProperty -Name ComputerName -value $view.Parent.ComputerName
-                Add-Member -Force -InputObject $view -MemberType NoteProperty -Name InstanceName -value $view.Parent.InstanceName
-                Add-Member -Force -InputObject $view -MemberType NoteProperty -Name SqlInstance -value $view.Parent.SqlInstance
-                Add-Member -Force -InputObject $view -MemberType NoteProperty -Name Database -value $db.Name
+                Add-Member -Force -InputObject $v -MemberType NoteProperty -Name ComputerName -value $v.Parent.ComputerName
+                Add-Member -Force -InputObject $v -MemberType NoteProperty -Name InstanceName -value $v.Parent.InstanceName
+                Add-Member -Force -InputObject $v -MemberType NoteProperty -Name SqlInstance -value $v.Parent.SqlInstance
+                Add-Member -Force -InputObject $v -MemberType NoteProperty -Name Database -value $db.Name
 
-                Select-DefaultView -InputObject $view -Property $defaults
+                Select-DefaultView -InputObject $v -Property $defaults
```

Comment:
Please revert this back to the original variable name.

Reply:
I had to add a parameter $View to select an individual view for the new function which conflicts with the $view variable here.  I can change this varaible to $sqlview to more closely align with get-dbatable if that would be preferable.

Reply:
That will work.

Reply:
Updated names